### PR TITLE
Update LICENSE information

### PR DESCRIPTION
**Pull request by christopherwharrop-noaa**
## Proposed changes

Update LICENSE information provided by Jeff Durachta from GFDL.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (Change which fixes an issue)
- [ ] New feature (Change which adds functionality)
- [x] Enhancement to existing functionality (non-breaking change which improves existing code or documentation)
- [ ] Breaking change (fix or feature that changes results and requires updates to test baselines)
- [x] Documentation Update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/NOAA-GSL/SENA-c_sw/blob/develop/CONTRIBUTING.md) document
- [x] I have ensured all my changes contribute to a common purpose
- [x] I have verified my change does not add debug code
- [x] I have verified my change does not add code that is commented out
- [x] I have verified unit tests pass locally with my changes
- [x] I have verified my changes adhere to style guidelines
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by christopherwharrop-noaa** [comment]
@robgonzalezpita - When you get a chance, can you please take a quick look at this?  Should be straightforward, but an approval is needed before merging can happen.  Please be on the lookout for a similar PR for the yppm.

**Comment by christopherwharrop-noaa** [comment]
@robgonzalezpita - I need an approval for this one as well.  Thank you.

**Review by robgpita — APPROVED** [review]

**Pull request merged**
